### ISSUE TITLE
Multiple feeds not processing

### ISSUE THREAD

**Issue by davemreed** [closemarketing/connector-gravityforms-mailerlite#4]
This is a great and simple plugin, thanks so much!

I am trying to use one Gravity Form to offer subscriptions to four different Mailerlite groups.

I added a checkbox field labeled “Newsletters” and have the four options there. I created an individual Mailerlite feed for each one, linked to the corresponding Group in Mailerlite. I enabled the conditional logic on each feed like this:

> Process this feed if ANY of the following match:
> Newsletters IS [Option X]

This is not working. If someone selects Option 1, that works. If they select Option 1 and, say, Option 3, they will be added to the Option 1 group, but not the Option 3 group. If someone selects only Option 2, they will not be added to that group.

Basically Option 1 works, the rest do not, no matter what the combination of selections is.

Option 1 – works
Option 2 – doesn’t work
Option 3 – doesn’t work
Option 4 – doesn’t work

Any idea if this is an error on my part, or an issue with the plugin?
